src/_includes/navbar.njk

```diff
@@ -76,8 +76,7 @@
                 </div>
               </div>
             {% else %}
-              {# TODO: item.name != 'Sponsors' is a hack till we get the sponsors page up#}
-              <a href="{% if item.isExternal %}{{ item.link }}{% elif item.link and item.link[0] == '#' and item.name != 'Sponsors' %}{{ item.link }}{% else %}{{ env.baseUrl }}{{ item.link }}{% endif %}"
+              <a href="{% if item.isExternal %}{{ item.link }}{% elif item.link and item.link[0] == '#' %}{{ item.link }}{% else %}{{ env.baseUrl }}{{ item.link }}{% endif %}"
```

Comment:
This navbar points to the sponsor section which stays on landing page.

The page you added is Why Sponsors section. This navbar shouldn't change at all.

Reply:
Is it though? The TODO says it was a hack.

Reply:
@farhaanbukhsh ^

Reply:
We should change the navbar link to point to the new page.

Reply:
Done

Reply:
yup it was a hack!  And this is the right approach